debian/naemon-livestatus.install

```diff
@@ -2,3 +2,5 @@
 /usr/lib/naemon/naemon-livestatus/livestatus.so
 /var/log/naemon
 /etc/naemon/module-conf.d/livestatus.cfg
+/usr/lib/python2.7/dist-packages/livestatus/__init__.py
+/usr/lib/python2.7/dist-packages/livestatus/livestatus.py
```

Comment:
I don't think this will work in general. Could this be generated dynamically from the rules file?